### PR TITLE
Sync with my repo (niico) - Up to 0.75.0

### PR DESCRIPTION
**Pull request by niicojs**

### CODE REVIEW

**Comment by niicojs** [comment]
This should include most of the other PR I guess

File: src/POGOProtos/Networking/Requests/RequestType.proto [hunk 1]
```diff
@@ -7,7 +7,7 @@ package POGOProtos.Networking.Requests;
 enum RequestType {
 	METHOD_UNSET = 0;                    // No implementation required
 	GET_PLAYER = 2;                      // Implemented [R & M]
-	GET_INVENTORY = 4;                   // Implemented [R & M]
+	GET_HOLO_INVENTORY = 4;              // Implemented [R & M]
```
Comment (AeonLucid) [inline src/POGOProtos/Networking/Requests/RequestType.proto:10]: Why?

File: src/POGOProtos/Networking/Requests/RequestType.proto [hunk 2]
```diff
@@ -87,9 +87,11 @@ enum RequestType {
...
 	USE_ITEM_MOVE_REROLL = 813;          // Implemented [R & M]
 	USE_ITEM_RARE_CANDY = 814;           // Implemented [R & M]
-	AWARD_FREE_RAID_TICKET = 815;
+	AWARD_FREE_RAID_TICKET = 815;        // Implemented [R & M]
 	REGISTER_PUSH_NOTIFICATION = 5000;
 	UNREGISTER_PUSH_NOTIFICATION = 5001;
 	UPDATE_NOTIFICATION_STATUS = 5002;
 	OPT_OUT_PUSH_NOTIFICATION_CATEGORY = 5003;
+	DOWNLOAD_GAME_MASTER_TEMPLATES = 5004;
+	GET_INVENTORY = 5005;
```
Comment (AeonLucid) [inline src/POGOProtos/Networking/Requests/RequestType.proto:96]: Why?

File: src/POGOProtos/Networking/Requests/RequestType.proto (hunk 1 above)
Comment (niicojs) [reply src/POGOProtos/Networking/Requests/RequestType.proto:10]: I don't know about the why, but that's what the real app is doing so I choose to keep sync with the real app.

Comment (AeonLucid) [reply src/POGOProtos/Networking/Requests/RequestType.proto:10]: Alright, that's good enough for me 👍

**Pull request merged**